tests/test_models.py

```diff
@@ -80,23 +93,25 @@ def test_create_csv(as_ops):
 # def test_filter_note_type():
 #     """Test filter_note_type function."""
 #     assert False
-#
-#
-# def test_replace_str():
-#     """Test replace_str function."""
-#     assert False
+
+
+def test_replace_str():
+    """Test replace_str function."""
+    old_string = 'The'
+    new_string = 'A'
+    field_value = 'The cow jumped over the moon'
+    new_value = models.replace_str(field_value, old_string, new_string)
+    assert new_string in new_value
 #
 #
 # def test_update_record():
 #     """Test update_record function."""
 #     assert False
-#
-#
-# def test_find_key():
-#     """Test find_key function."""
-#     assert False
-#
-#
-# def test_get_aos_for_resource():
-#     """Test get_aos_for_resource function."""
-#     assert False
+
+
+def test_find_key():
+    """Test find_key function."""
+    nest_dict = {'a': {'b': False, 'c': True}, 'c': True}
```

Comment:
Here you might also want to check the length of `keys` to ensure that it found two instances.

Reply:
Good call